### PR TITLE
docs(DEVELOPER): JDK is needed for protractor to run selenium

### PR DESCRIPTION
**Pull request by alfonso-presa**
Java's JDK is needed for selenium standalone server, right?

### CODE REVIEW

**Comment by zoechi** [comment]
I haven't used selenium with Angular yet but I would assume this would do as well https://github.com/SeleniumHQ/docker-selenium

**Comment by alfonso-presa** [comment]
Sorry for the lack of context. I meant that it was needed to run selenium standalone server during the angular's build process for e2e testing.

**Comment by jeffbcross** [comment]
@alfonso-presa could you add a note to add context about this being a requirement for Selenium Standalone Server?

**Comment by alfonso-presa** [comment]
Done!

File: DEVELOPER.md [hunk 1]
```diff
@@ -47,6 +47,8 @@ following products on your development machine:
 
 * [Bower](http://bower.io/).
 
+* [Java Development Kit](http://www.oracle.com/technetwork/es/java/javase/downloads/index.html) wich is used
```
Comment (gkalpak) [inline DEVELOPER.md]: _wich_  -->  _which_

Comment (alfonso-presa) [inline DEVELOPER.md]: Ops!

**Pull request closed without merging**